data/categories.txt

```diff
@@ -2850,7 +2850,7 @@ highway-primary|highway-primary_link|highway-residential|highway-secondary|highw
 en:street|st|str|ave|road|rd
 ru:улица|переулок|проезд|тракт|проспект|ул
 uk:вулиця|провулок|проїзд|проспект|бульвар|вул
-de:Strasse|Weg|Avenue|Gasse|Str
+de:Straße|Weg|Boulevard|Gasse
```

Comment:
Str should match it anyway, I suppose it is also hard-coded somewhere else in the search engine.
Our search should correctly handle ß (you can probably test it by placing categories.txt into app data folder on iTunes or MapsWithMe folder on android).

> On Feb 1, 2016, at 19:15, malenki <notifications@github.com> wrote:
> 
> In data/categories.txt <https://github.com/mapsme/omim/pull/1735#discussion_r51438590>:
> 
> > @@ -2850,7 +2850,7 @@ highway-primary|highway-primary_link|highway-residential|highway-secondary|highw
> >  en:street|st|str|ave|road|rd
> >  ru:улица|переулок|проезд|тракт|проспект|ул
> >  uk:вулиця|провулок|проїзд|проспект|бульвар|вул
> > -de:Strasse|Weg|Avenue|Gasse|Str
> > +de:Straße|Weg|Boulevard|Gasse
> I thought that searching for Str would match on Straße as well. Is that so?
> 
> Does the program translate "ß" to "ss" and similar stuff? Else I would have to add back "Strasse" … which is just the way of writing "Straße" when people have no "ß" at hand (that is nearly everybody outside Austria/Germany).
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/1735/files#r51438590>.
>